kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseSiteService.java

```diff
@@ -1493,6 +1494,8 @@ public void removeSite(Site site, boolean isHardDelete) throws PermissionExcepti
 				if(!site.isSoftlyDeleted()) {
 					site.setSoftlyDeleted(true);
 					save(site);
+					// Trigger site soft-delete event
```

Comment:
I don't think you need this comment, personally. A sakai dev should know what the eventTrackingService.post method does. Up to you, though.